### PR TITLE
Draft grant standard

### PR DESCRIPTION
**Pull request by JFickel**
allô!

### CODE REVIEW

File: EIPS/eip-draft_grant_standard.md [hunk 1]
```diff
@@ -9,50 +9,50 @@ category: ERC
 created: 2019-04-30
 ---
 
-## Simple Summary
```
Comment (ArnaudBrousseau) [inline EIPS/eip-draft_grant_standard.md:12]: We should probably leave this in? Seems like it's something expected for each EIP? ("abstract" is a 200 words summary, "simple summary" should just be sentence)

**Comment by ArnaudBrousseau** [comment]
Anyway, seems good to merge! We can bring back Simple Summary later, no big deal.

File: EIPS/eip-draft_grant_standard.md (hunk 1 above)
Comment (JFickel) [reply EIPS/eip-draft_grant_standard.md]: Idk. It felt redundant. I was looking at the ERC-20 proposal from way back, and he took it out

https://github.com/ethereum/eips/issues/20

Comment (JFickel) [reply EIPS/eip-draft_grant_standard.md:12]: We can do that. Idk. It felt redundant

**Pull request merged**